admin/create/country/country.controller.js

```diff
@@ -6,38 +6,55 @@
         .module('admin')
         .controller('CreateCountryController', CreateCountryController);
 
-    function CreateCountryController ($log, $http, NotificationService, Endpoints) {
+    function CreateCountryController($log, $routeSegment, NotificationService, countriesProm, CreatePlacesService) {
         const vm = this;
 
         vm.countries = [];
-        vm.newCountry = '';
+        vm.newCountry = {};
 
         vm.createNewCountry = createNewCountry;
 
         activate();
 
         ////
 
-        function activate () {
+        function activate() {
             $log.debug('Init CreateCountryController ...');
             getCountries();
         }
 
         function getCountries() {
-            $http.get(Endpoints.COUNTRIES).then((res) => {
-               
-                if(res.data.success) {
-                    vm.countries = res.data.data;
+            countriesProm.$promise.then((res) => {
```

Comment:
Some people might not understant what is "Prom". Use whole word, like `countriesPromise`